starter-code/contact-app/src/styles.css

```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+body {
+  padding: 40px;
```

Comment:
You are aplying the exact same style to every component. In that case, maybe it's better no repeat it.